--- nforenum-4.0.0-source/Makefile~	2010-08-14 21:47:40.000000000 +0200
+++ nforenum-4.0.0-source/Makefile	2010-08-17 09:27:49.050660672 +0200
@@ -33,7 +33,7 @@
 ifeq ($(ISCYGWIN),1)
 FLAGS = -O1 -I $(BOOST_INCLUDE)
 else
-FLAGS = -O1 -idirafter$(BOOST_INCLUDE)
+FLAGS = -idirafter$(BOOST_INCLUDE)
 endif
 FLAGS += -Wall -Wno-uninitialized -Wsign-compare -Wwrite-strings -Wpointer-arith -W -Wno-unused-parameter -Wformat=2
 FLAGS += -D_FORTIFY_SOURCE=2
@@ -131,7 +131,7 @@
 
 $(NFORENUM): $(NFORENUMSRC:%.cpp=objs/%.o)
 	$(_E) [LD] $@
-	$(_C)$(CXX) -o $@ $(CXXFLAGS) $^ $(LDOPT)
+	$(_C)$(CXX) $(CXXFLAGS) $(LDOPT) -o $@ $^
 
 
 clean:
@@ -165,7 +165,7 @@
 
 objs/%.o : src/%.cpp Makefile
 	$(_E) [CPP] $@
-	$(_C)$(CXX) -c -o $@ $(CXXFLAGS) -MMD -MF $@.d -MT $@ $<
+	$(_C)$(CXX) $(CXXFLAGS) -c -o $@ -MMD -MF $@.d -MT $@ $<
 
 # On some installations a version.h exists in /usr/include. This one is then
 # found by the dependency tracker and thus the dependencies do not contain
